crates/uv-virtualenv/src/virtualenv.rs

```diff
@@ -143,21 +146,58 @@ pub(crate) fn create(
     // Create a `.gitignore` file to ignore all files in the venv.
     fs::write(location.join(".gitignore"), "*")?;
 
+    let mut using_symlink_path = false;
+    let executable_target = if interpreter.is_standalone() {
+        if let Some(symlink_path) =
+            interpreter.maybe_symlink_path_from_base_python(base_python.as_path())
+        {
+            using_symlink_path = true;
+            symlink_path
+        } else {
+            base_python.clone()
+        }
+    } else {
+        base_python.clone()
+    };
+
     // Per PEP 405, the Python `home` is the parent directory of the interpreter.
-    let python_home = base_python.parent().ok_or_else(|| {
-        io::Error::new(
-            io::ErrorKind::NotFound,
-            "The Python interpreter needs to have a parent directory",
-        )
-    })?;
+    // For standalone interpreters, this `home` value will include a symlink directory
+    // on Unix or junction on Windows to enable transparent Python patch upgrades.
+    let python_home = executable_target
+        .parent()
+        .ok_or_else(|| {
+            io::Error::new(
+                io::ErrorKind::NotFound,
+                "The Python interpreter needs to have a parent directory",
+            )
+        })?
+        .to_path_buf();
+    let python_home = python_home.as_path();
 
     // Different names for the python interpreter
     fs::create_dir_all(&scripts)?;
     let executable = scripts.join(format!("python{EXE_SUFFIX}"));
 
+    if using_symlink_path {
```

Comment:
Is there a reason this can't be performed above instead of tracking this state in a boolean?

Reply:
Fixed